source/faq.txt

```diff
@@ -171,6 +171,28 @@ your classpath, check the following items in your environment:
    :ref:`API documentation <java-api-landing>`.
 
 
+How do I prevent the "com.mongodb.MongoSecurityException" error?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Your application may throw this exception if you attempt to connect to
+a MongoDB deployment by specifying invalid or incorrectly formatted credentials.
+
+If you receive this error when you attempt to connect to a MongoDB deployment,
+check the following items in your code:
+
+- The connection URI corresponds to the correct MongoDB deployment.
+  To learn more about setting your connection URI, see :ref:`connection-uri`.
+
+- The credentials for the authentication mechanism that you specified are
```

Comment:
S: On line 191, you leave out "that" in the first sentence. Suggest being consistent across list items.